### DISCUSSION TITLE
Lazer Play Count Cheese

### DISCUSSION THREAD

**Discussion by ChaBoiNeil** [ppy/osu#19355] [Q&A] [answered]
### Type

Game behaviour

### Bug description

This might need to be fine tuned. (Video shows the issue.)

All it takes to get a 'single play count' for a map only requires 1 point. This is probably a nit-picking issue, but I would like to bring it up since it is a thing.

### Screenshots or videos

https://user-images.githubusercontent.com/103234709/180662133-df0bf01e-acd1-4086-960a-45f42b76a7a2.mp4

### Version

2022.723.0-lazer

### Logs

N/A

**Comment by peppy** [comment, marked as answer]
All statistics and scores given current are subject to change. We'll track this over at https://github.com/ppy/osu-queue-score-statistics/issues/63.
